docs/calamari/CommunityCollatorsProgram.md

```diff
@@ -0,0 +1,287 @@
+    2022-01-25 14:15:09 [Relaychain] 💤 Idle (9 peers), best: #570132 (0x8929…0353), finalized #570130 (0xb471…6943), ⬇ 5.2kiB/s ⬆ 3.3kiB/s
+    ```
+
+    If you have 0 peers on the relaychain and/or your node is failing to verify new  blocks, ensure your node’s clock is accurate, e.g. by forcing a sync with an NTP timeserver 
+
+    ```jsx
+    2022-01-25 13:47:49 [Relaychain] 💔 Verification failed for block 0xbd3bf6d173daad66fef2325d3a1bc3141de630519ad0c390ccafec9d24faeba7 received from peer: 12D3KooW9tzdaAsSfzY97cPwSFx8ALxci1sppCHJaSddDrxs9FCp, "Header 0xbd3bf6d173daad66fef2325d3a1bc3141de630519ad0c390ccafec9d24faeba7 rejected: too far in the future" rification failed for block 0xf8
+    ```
+    
+6. Bond your account id with session keys which are generated from `step-3`.
+
+    ![bond-keys](images/bond-session-keys.png)
+
+    Both params should be the same.
+
+7. Fill the [collator application](https://docs.google.com/forms/d/e/1FAIpQLScizDDMq7jWeOPVVEMr3EY_Z6N6ugdkL8aKgAbZ9lAJX6DEOQ/viewform) form. If approved, Calamari council will submit a motion to promote you as a candidate.
+> Tips: Candidate doesn’t mean your node is collator. For example, if current candidates spots is 3, and some other candidates occupy all spots, you’re the 4th, you have to wait until there’s a node is unregistered or more collator spots are open by council.
+
+8. After ~2 sessions(about `6` to `12` hours) pass, you will see blocks produced from your collator. 
+
+    ![block-production](images/block-production.png)
+
+### Deploy a collator on Ubuntu using docker
+
+1. Get latest calamari image.
+
+    ```bash
+    docker pull mantanetwork/calamari:latest
+    ```
+
+2. Start a container. All your data will be save to `host_path`.
+
+    > If you want to start collator with snapshot, download snapshot from [here](#download-snapshot),
+    > follow this [section](#docker) to deploy node with snapshot.
+    ```bash
+    docker run \
+    -it \
+    -p 9933:9933 \
+    -p 30333:30333 \
+    -v host_path:/container_path \
+    --name your_container_name \
+    mantanetwork/calamari:latest \
+    --base-path /container_path/data \
+    --keystore-path /container_path/keystore \
+    --name your_collator_name \
+    --rpc-cors all \
+    --collator \
+    --rpc-methods=unsafe \
+    --unsafe-rpc-external
+    ```
+
+    Examples of these name and paths:
+
+    `host_path:/container_path` => `~/my-calamari-db:/calamari`
+
+    `your_collator_name` => `Community-Collator-1`
+
+    Ensure you can see a line of log like this:
+
+    ```bash
+    👤 Role: AUTHORITY
+    ```
+
+    ![collator-log](images/collator-log.png)
+
+3. Generate session keys.
+
+    ```bash
+    curl -H 'Content-Type: application/json' --data '{ "jsonrpc":"2.0", "method":"author_rotateKeys", "id":1 }' http://localhost:9933
+    ```
+
+4. Now stop your current running container for security reason, remove two params.
```

Comment:
Maybe this will elaborate better
[https://www.notion.so/How-to-add-remove-a-collator-Community-e5a7b8e595eb4515b3fc0c581668c334#5cd57d091d1c4968b4deec61caf0738e](url)

Reply:
The container is given a name, it's identical.